fluentd/out_syslog.rb

```diff
@@ -92,7 +92,14 @@ def emit(tag, es, chain)
                           tag[0..31] # tag is trimmed to 32 chars for syslog_protocol gem compatibility
                       end
         packet = @packet.dup
-        packet.content = record[@payload_key]
+        if @use_record && (defined? record["kubernetes"]) && (!record["kubernetes"].nil?)
```

Comment:
I think you should use `key?` instead of `defined?`
`record.key?('kubernetes')`
http://ruby-doc.org/core-2.0.0/Hash.html#method-i-key-3F